ibc.ts/src/relayer/index.ts

```diff
@@ -120,7 +142,8 @@ async function updateLightClient({
         );
     }
     let currentBlockNumber = clientState!.data!.number;
-    while (currentBlockNumber < counterpartyChainHeight) {
+    // We can't get the best block's IBC header
```

Comment:
I think `FIXME` is needed here, because there is `FIXME` in `compose_header()` for the same problem.